File: Java/Q19.java
class Main {
    public static void main(String[] args) {
        int fuel_choice = 2;
        switch (fuel_choice) {
            case 1: System.out.println("You have chosen: Petrol"); break;
            case 2: System.out.println("You have chosen: Diesel"); break;
            case 3: System.out.println("You have chosen: Electric"); break;
            default: System.out.println("Invalid choice.");
        }
    }
}

File: CPP/Q19.cpp
#include <iostream>

int main() {
    int fuel_choice = 2;
    switch (fuel_choice) {
        case 1: std::cout << "You have chosen: Petrol" << std::endl; break;
        case 2: std::cout << "You have chosen: Diesel" << std::endl; break;
        case 3: std::cout << "You have chosen: Electric" << std::endl; break;
        default: std::cout << "Invalid choice." << std::endl;
    }
    return 0;
}